contributors/design-proposals/scheduler_extender.md

```diff
@@ -3,7 +3,7 @@
 There are three ways to add new scheduling rules (predicates and priority
 functions) to Kubernetes: (1) by adding these rules to the scheduler and
 recompiling (described here:
-https://github.com/kubernetes/kubernetes/blob/master/docs/devel/scheduler.md),
+https://github.com/kubernetes/community/blob/master/contributors/devel/scheduler.md),
```

Comment:
I'd prefer the format as

recompiling, [described here](https://github.com/kubernetes/community/blob/master/contributors/devel/scheduler.md)